lib/auth/auth.go

```diff
@@ -1740,6 +1749,7 @@ func (a *Server) ExtendWebSession(req WebSessionReq, identity tlsca.Identity) (t
 
 		roles = append(roles, newRoles...)
 		roles = apiutils.Deduplicate(roles)
+		accessRequests = apiutils.Deduplicate(append(accessRequests, req.AccessRequestID))
```

Comment:
`AccessRequestID` may be `""`.  The append should be conditional.

Reply:
We're in `if req.AccessRequestID != "" {` here